apps/cowswap-frontend/src/legacy/hooks/useWrapCallback.ts

```diff
@@ -57,7 +58,7 @@ export async function wrapUnwrapCallback(
     wrapAnalytics('Send', operationMessage)
 
     const wrapUnwrap = isNativeIn ? wrapContractCall : unwrapContractCall
-    const txReceipt = await wrapUnwrap(wethContract, amountHex)
+    const txReceipt = await wrapUnwrap(wethContract, amountHex, chainId)
```

Comment:
Here we not ensuring chainId and wethContract are in sync? we trust he params are coherent and there's no race condition?